docs/TravisFileExplained.md

```diff
@@ -31,15 +31,15 @@ cache:
 # listed here will create a separate build and run the tests against that
 # version of PHP.
 php:
- - 7.2
  - 7.3
  - 7.4
+ - 8.0
 
 # This section sets up the environment variables for the build.
 env:
  global:
 # This line determines which version branch of Moodle to test against.
-  - MOODLE_BRANCH=MOODLE_39_STABLE
+  - MOODLE_BRANCH=MOODLE_311_STABLE
```

Comment:
Same as per comment about GHA, I suggest to add same changes (version and branch matrices) to travis.dist.yml.

Reply:
New mini-commit (https://github.com/moodlehq/moodle-plugin-ci/pull/102/commits/bfe3366933791c49bebcb26a89b6dbe93653d9ec) makes the dist files to match the Explained docs.